common/dataSort.js

```diff
@@ -0,0 +1,90 @@
+const R = require('ramda')
+
```

Comment:
I don't think it's a good idea to mix dataSort object (which is a UI model object) in common folder. can it be moved back where it was?
You can then move the ```getSortPreparedStatement``` into the TableViewQuery or create a module that does the conversion into the same folder of TableViewQuery

Reply:
ok  let's put this in common/surveyRdb

Reply:
Ok